StudCity.Frontend/src/components/Message/index.jsx

```diff
@@ -27,10 +27,12 @@ export default function Message({ userId, content, when, fullName, image, id })
 				</div>
 			) : (
 				<div className={`${allContainer} mr-auto ml-10`}>
-					<Avatar
-						className={`${avatarStyle}`}
-						src={image}
-					/>
+					<div className="h-full flex mt-2 mb-auto">
+						<Avatar
+							className={`${avatarStyle} `}
```

Comment:
I'm guessing you written like this for the future use, if not then simply write: className={avatarStyles}